cnf/ext/central.mvn

```diff
@@ -70,6 +70,7 @@ org.littleshoot:littleproxy:1.1.2
 io.netty:netty-all:4.1.0.Final
 
 com.google.guava:guava:19.0
+com.google.guava:guava:21.0
```

Comment:
I think you also need to modify the bnd.bnd file in biz.aQute.bnd.reporter to add a version range

    com.google.guava;version="[18.0,19)",\